modules/search-solr-client/src/main/scala/io/renku/search/solr/client/SearchSolrClientImpl.scala

```diff
@@ -39,10 +41,16 @@ private class SearchSolrClientImpl[F[_]: Async](solrClient: SolrClient[F])
       limit: Int,
       offset: Int
   ): F[QueryResponse[Project]] =
-    val solrQuery = QueryInterpreter(query)
-    logger.debug(s"Query: ${query.render} ->Solr: $solrQuery") >>
-      solrClient
-        .query[Project](QueryData.withChildren(QueryString(solrQuery, limit, offset)))
+    for {
+      solrQuery <- interpreter.run(query)
+      _ <- logger.debug(s"Query: ${query.render} ->Solr: $solrQuery")
+      res <- solrClient
+        .query[Project](
+          QueryData
+            .withChildren(QueryString(solrQuery.query.value, limit, offset))
```

Comment:
I'm guessing, as we took out the nesting, we might not the `withChildren` here anymore.